.github/workflows/rust.yml

```diff
@@ -61,15 +61,7 @@ jobs:
 
   test:
     name: ${{ matrix.build }}
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        build: [Linux, macOS]
-        include:
-          - build: Linux
-            os: ubuntu-latest
-          - build: macOS
-            os: macos-latest
+    runs-on: ubuntu-latest
```

Comment:
I guess these changes won't solve the problem of the long CI execution because it decreases the number of jobs by one only. I would combine check fmt with clippy but leave the matrix as is.

Reply:
Do not agree at all. Proof: https://github.com/aurora-is-near/borealis-engine-lib/actions/runs/6492905035
and after: https://github.com/aurora-is-near/borealis-engine-lib/actions/runs/6493330374